### PR TITLE
Update MMS NoW ETLs

### PR DESCRIPTION
**Pull request by nayr974**

### CODE REVIEW

File: tasks/now-etls/mms_now_import.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def ETL_MMS_NOW_schema(connection, tables):
...
+            r"""SELECT now.msg_id as messageid, now.cid as mms_cid, now.mine_no as minenumber, now.apl_dt as submitteddate, now.lat_dec, now.lon_dec, now.multi_year_ind, now.multi_year_area_ind, now.str_dt as ProposedStartDate, now.end_dt as ProposedEndDate, now.site_desc as SiteDirections, now.prpty_nm as NameOfProperty, now.apl_typ, CASE WHEN vw.status = 'ACCEPTED' THEN 'Pending Verification' WHEN vw.status = 'APPROVED' THEN 'Approved' WHEN vw.status = 'CLIENT DELAYED' THEN 'Client delayed' WHEN vw.status = 'GOV. ACTION REQ.' THEN 'Govt. Action Required' WHEN vw.status = 'NO PERMIT REQUIRED' THEN 'No Permit Required' WHEN vw.status = 'PENDING APPROVAL' THEN 'Pending Approval' WHEN vw.status = 'PERMIT CLOSED' THEN 'Permit Closed' WHEN vw.status = 'RECEIVED' THEN 'Received' WHEN vw.status = 'REFERRAL COMPLETE' THEN 'Referral Complete' WHEN vw.status = 'REFERRED' THEN 'Referred' WHEN vw.status = 'REJECTED' THEN 'Rejected' ELSE NULL END AS status from mms.mmsnow now JOIN mms.mmsnow_vw vw ON now.cid=vw.cid"""
```
Comment (exzizt) [inline tasks/now-etls/mms_now_import.py:47]: - 'ACCEPTED' THEN 'Pending Verification'?

- Client delayed --> Client Delayed

Comment (mmumby) [reply tasks/now-etls/mms_now_import.py:47]: We decided yesterday that 'PERMIT CLOSED'  -> Approved (It was once approved, and then they mixed in the permit logic and set to permit closed when the bond is returned)

Comment (mmumby) [reply tasks/now-etls/mms_now_import.py:47]: We also don't have  'Pending Approval' as an option any longer,
 'PENDING APPROVAL' THEN 'Pending Verification' ? 
 Users will have to import and start the application from scratch within CORE

Comment (nayr974) [reply tasks/now-etls/mms_now_import.py:47]: "Client delayed" matches the value in our code table. Accepted -> pending verification per Ashley's direction. Updated permit closed to approved.

Comment (exzizt) [reply tasks/now-etls/mms_now_import.py]: It was changed to have consistent casing with my PR yesterday.

Comment (nayr974) [reply tasks/now-etls/mms_now_import.py]: Ugh, FINE Luke

Comment (exzizt) [reply tasks/now-etls/mms_now_import.py]: ;)

**Review by exzizt — APPROVED** [review]
💥

**Pull request closed without merging**